Source/Core/Core/DSP/Jit/DSPEmitter.cpp

```diff
@@ -333,7 +333,7 @@ void DSPEmitter::Compile(u16 start_addr)
   {
     m_block_links[start_addr] = m_block_link_entry;
 
-    for (u16 i = 0x0000; i < 0xffff; ++i)
+    for (size_t i = 0; i < 0xffff; ++i)
```

Comment:
This loop range looks odd, being basically `MAX_BLOCKS - 1` (also, maybe `m_unresolved_jumps.size()` instead?)